apps/public/src/app/(general)/samenvatting/utils.ts

```diff
@@ -64,6 +66,46 @@ export const getAdditionalQuestionsSummary = async (meldingId: string, token: st
   }
 }
 
+export const getAttachmentsSummary = async (label: string, meldingId: string, token: string) => {
+  const { data, error } = await getMeldingByMeldingIdAttachmentsMelder({
+    path: { melding_id: parseInt(meldingId, 10) },
+    query: { token },
+  })
+
+  if (error) return { error: handleApiError(error) }
+
+  if (!data) return { error: 'Attachments data not found' }
+
+  let downloadError: string | undefined
+
+  const attachments = await Promise.all(
+    data.map(async (attachmentDetails) => {
+      const { data, error, response } = await getMeldingByMeldingIdAttachmentByAttachmentIdDownload({
+        path: { melding_id: parseInt(meldingId, 10), attachment_id: attachmentDetails.id },
+
+        query: { token, type: 'thumbnail' },
+      })
+
+      const contentType = response.headers.get('content-type')
+
+      if (error) {
+        downloadError = handleApiError(error)
+      } else if (!data) {
+        downloadError = 'Attachment data not found'
+      }
+
+      return {
+        file: data as Blob,
+        meta: { originalFilename: attachmentDetails.original_filename, contentType: contentType! },
```

Comment:
Maybe we should add a comment here, on why we're returning a blob instead of a file. So we don't have to figure out the reason again in a years time.